digdag-standards/build.gradle

```diff
@@ -15,7 +15,17 @@ dependencies {
     compile 'com.sun.mail:javax.mail:1.5.6'   // 'com.sun.mail:smtp' doesn't work because enabling mail.debug property throws java.lang.NoClassDefFoundError: com/sun/mail/util/MailLogger
 
     // td
-    compile 'com.treasuredata.client:td-client:0.7.36'
+    compile ('com.treasuredata.client:td-client:0.8.6') {
+        // digdag-tests depends on okhtttp3 3.4.1, 
+        // but td-client:0.8.6 depends on okhttp3 3.9.0 for now.
+        // To avoid affecting to digdat-tests, I excluded okhttp3 here.
+        exclude group: 'com.squareup.okhttp3', module: 'okhttp'
+        exclude group: 'com.squareup.okhttp3', module: 'okhttp-urlconnection'
+        // digdag depends on guava 0.19.0
+        exclude group: 'com.google.guava', module: 'guava'
+        exclude group: 'com.google.inject', module: 'guice'
+        exclude group: 'ccom.google.inject.extensions', module: 'guice-multibindings'
```

Comment:
ccom..

Reply:
😢

Reply:
I will remove `guice` and `guice-multibindings` because these are not necessary exclusion.
When I tried `mvn validate` without `guice` and `guice-multibindings` , it succeeded.